### PR TITLE
Add documentation and apply linter advice

### PR DESCRIPTION
**Pull request by DerekStrickland**
## Summary

Overview of changes

### Added

List new features or components. Include a screenshot for new visual elements.

### Changed

List changes in existing functionality or design.
If the change was visual, include a comparison screenshot showing the before and after the visual change.

### Deprecated

List once-stable features or components to be deprecated in this PR.

### Removed

List deprecated features or components removed in this PR.

### Fixed

List any bug fixes.

## How to test

Instructions on how to test the changes. This is not exhaustive list of ways you should test this PR.

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
+components = [ "rustfmt", "cargo", "clippy" ]
```
Comment (qtpeters) [inline rust-toolchain.toml:3]:
```suggestion
components = [ "rustfmt", "cargo", "clippy" ]

```

**Review by qtpeters — APPROVED** [review]
One file with no newline on the end, but otherwise great!

File: rust-toolchain.toml (hunk 1 above)
Comment (DerekStrickland) [reply rust-toolchain.toml:3]: Incoming in next commit.

**Review by talentedmrjones — APPROVED** [review]
:shipit:

**Pull request merged**